core-container/entry.sh

```diff
@@ -244,8 +257,8 @@ function run_init() {
     echo "--< Starting init >--"
     wait_kcp_ready
     set_provider_adapters
-    kubestellar init --local-kcp false --ensure-imw $ENSURE_IMW --ensure-wmw $ENSURE_WMW
-    kubectl ws root
+    # sleep infinity # EFFI Debugging
+    kubestellar init --local-kcp false --ensure-imw $ENSURE_IMW --ensure-wmw $ENSURE_WMW 
```

Comment:
I'm a little puzzled to why we have wait_kcp_ready and a --local-kcp  flags at this point in code  where we supposed to be space only code